### DISCUSSION TITLE
Problem with PS1

### DISCUSSION THREAD

**Discussion by gnrgui** [eduardofilo/RG350_adam_image#124] [SimpleMenu topics]
Hi! I'm having a problem when I try to enter the PS1 list of games, the system just hangs on a loading just like the image:
![20211214_224749](https://user-images.githubusercontent.com/20758034/146108277-926c6448-8a2e-4822-88d6-425a6f90bf3a.jpg)
My games worked fine before, today I've added some games from other systems and this began to happen. I've tried to left just one PS1 game in the folder, but the same thing occurs.
If I open the PS1 OPK via the commander app, the games load and work fine. The problem really seems to be the game list on the simplemenu.
Thanks in advance.

**Comment by eduardofilo** [comment]
Hi. Try removing the file `/media/data/local/home/.simplemenu/last_state.sav`. You can do it throught Commander for instance. You will loose your SimpleMenu settings (like theme selected and so), but they are just a few adjustments.

**Reply by AndreZoro** [reply]
Had the same issue and this solved it.
